cds_migrator_kit/videos/weblecture_migration/load/helpers.py

```diff
@@ -61,10 +74,25 @@ def copy_file_to_bucket(bucket_id, file_path):
 
         # Copy file to storage.
         shutil.copy2(file_path, full_path)
+        
+        # Control if the file copied succesfully
+        file_checksum = file_storage.checksum()
+        with open(file_path, "rb") as file_stream:
```

Comment:
wouldn't that slow things down as you open the file stream to read the whole file again?